install.php

```diff
@@ -133,7 +133,7 @@ function ArraySearchRecursive($Needle,$Haystack,$NeedleKey="",$Strict=false,$Pat
 	$usePeople=($result->rowCount()>0 && !ArraySearchRecursive('fac_People',$test))?($result->rowCount()>0 && !ArraySearchRecursive('fac_people',$test))?false:true:true;
 
 	// New install so create a user
-	require_once("customers.inc.php");
+	require_once("classes/People.class.php");
```

Comment:
FYI, we haven't even touched the installer yet.  That's always the last
file we update for each new release, along with the create.sql file.  You
are correct in that for release 4.3, customers.inc.php will not exist.  If
people follow the best practices for upgrades, it won't even be there for
an upgrade, because you should untar the new release into a separate
folder, not on top of the existing installation.

Scott


On Tue, Jun 14, 2016 at 10:40 AM, unonu <notifications@github.com> wrote:

> In install.php
> <https://github.com/samilliken/openDCIM/pull/784#discussion_r66983718>:
>
> > @@ -133,7 +133,7 @@ function ArraySearchRecursive($Needle,$Haystack,$NeedleKey="",$Strict=false,$Pat
> >  	$usePeople=($result->rowCount()>0 && !ArraySearchRecursive('fac_People',$test))?($result->rowCount()>0 && !ArraySearchRecursive('fac_people',$test))?false:true:true;
> >
> >  	// New install so create a user
> > -	require_once("customers.inc.php");
> > +	require_once("classes/People.class.php");
>
> I guess what I'm trying to ask is:
> require_once("classes/People.class.php"); will work fine on both installs
> *and* upgrades whereas require_once("customers.inc.php"); will exit on
> new installs, right? And line 141
> require_once("installer.userconversion.inc.php"); will take care of
> loading the User class if it is an older install (again regardless of new
> or update.)
>
> Also I may not have made it clear that customers.inc.php doesn't exist
> anymore, so if that line needs to stay, there should also be a check for
> its existence.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/samilliken/openDCIM/pull/784/files/fe9983628d176526a103878d9f9e66c2c21f66c9#r66983718>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABk8e0sAv_PXIVgFsrC69tQNJ0oYakXXks5qLr1LgaJpZM4I1VHT>
> .
>